assets/js/blocks/featured-product/index.js

```diff
@@ -50,6 +50,14 @@ registerBlockType( 'woocommerce/featured-product', {
 			default: true,
 		},
 
+		/**
+		 * A fixed height for the block.
+		 */
+		height: {
+			type: 'number',
+			default: 500,
```

Comment:
It'd be nice if this used the same variable as above so it's only in one place.

Reply:
I agree these should be extendable, but technically these don't have to be related – MIN_HEIGHT could be a smaller value if we want to allow folks to shrink down the block.